paper-autocomplete-chips.html

```diff
@@ -143,6 +160,30 @@
 					value: 0
 				},
 
+				/**
+				 * Minimum limit of selected items, set to 0 to disable.
+				 */
+				 minSelected: {
```

Comment:
`<input>` attributes are `min` / `max` - we might want to use the same?

Reply:
Renamed them to min and max.